config/rootfs/debos/scripts/buster-libcamera.sh

```diff
@@ -41,9 +41,15 @@ echo '{  "tests_suites": [' >> $BUILDFILE
 ########################################################################
 
 LIBCAMERA_URL=https://git.linuxtv.org/libcamera.git
+COMMIT_ID=0d50a04cc918e5122d1b22201cc949c2cfd337a5
```

Comment:
I actually think it's better to leave it hard-coded in the `git revert` command, as this looks like the commit id we're using to build libcamera rather than a commit to revert.

Reply:
done